README.md

```diff
@@ -76,14 +76,83 @@ FUTO has generously awarded Freenet a Legendary Grant to support Locutus develop
 
 ### Supporting Locutus
 
-If you are in a position to fund our continued efforts please contact us on [twitter](https://twitter.com/FreenetOrg) or by email at 
+If you are in a position to fund our continued efforts please contact us on [twitter](https://twitter.com/FreenetOrg) or by email at
 *ian at freenet dot org*.
 
 ## License
 
 This project is licensed under either of:
 
-- Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
-  http://www.apache.org/licenses/LICENSE-2.0)
-- MIT license ([LICENSE-MIT](LICENSE-MIT) or
-  http://opensource.org/licenses/MIT)
+* Apache License, Version 2.0, ([LICENSE-APACHE](LICENSE-APACHE) or
+  <http://www.apache.org/licenses/LICENSE-2.0>)
+* MIT license ([LICENSE-MIT](LICENSE-MIT) or
+  <http://opensource.org/licenses/MIT>)
+
+## Building with Docker Images
```

Comment:
To keep the readme lean, maybe we should add a new chapter to the book for this and just reference that chapter here. @sanity opinion?

Reply:
A new chapter makes sense to me.  Let me know what you want me to do.